### PR TITLE
Feat gameinfo logic

### PR DESCRIPTION
**Pull request by sagargoel1911**

### CODE REVIEW

File: src/screens/GameInfo/GameInfo.tsx [hunk 1]
```diff
@@ -70,6 +73,29 @@ const styles = StyleSheet.create({
...
+
+	const { username } = useAppSelector(
+		(state) => ({
+			username: state.persistedUserData.username,
+		}),
+		shallowEqual,
+	);
+
+	const toggle_color = () => {
+		set_player_color((current_color) => (current_color === 'White' ? 'Black' : 'White'));
```
Comment (atulpatel-03) [inline src/screens/GameInfo/GameInfo.tsx:88]: add these colors in constant

File: src/screens/SignUp/SignUp.tsx [hunk 2]
```diff
@@ -13,9 +13,21 @@ const styles = StyleSheet.create({
...
+	password: string;
+	username: string;
+};
+
 const SignUp = () => {
 	const [nav, set_nav] = useState<any>();
-	const methods = useForm<any>();
+	const methods = useForm<FormValues>({
+		defaultValues: {
+			email: '',
```
Comment (atulpatel-03) [inline src/screens/SignUp/SignUp.tsx:26]: make form constants

File: src/screens/Login/components/Content.tsx [hunk 3]
```diff
@@ -58,8 +58,18 @@ const styles = StyleSheet.create({
...
+type FormValues = {
+	id: string;
+	password: string;
+};
+
 const Content = () => {
-	const methods = useForm<any>();
+	const methods = useForm<FormValues>({
+		defaultValues: {
+			id: '',
```
Comment (atulpatel-03) [inline src/screens/Login/components/Content.tsx:69]: make form constant

File: src/screens/SignUp/components/Header.tsx [hunk 4]
```diff
@@ -25,11 +25,15 @@ const styles = StyleSheet.create({
 	},
 });
 
-const Header = (props: any) => {
+interface Props {
+	nav: any;
```
[the remark this answers is not in the document]
Comment (sagargoel1911) [reply src/screens/SignUp/components/Header.tsx:29]: This prop is for the signup stack
The useNavigation app gives the navigation for the App Stack

**Pull request merged**